app/src/release/res/values/google_maps_api.xml

```diff
@@ -17,6 +17,6 @@
     string in this file.
     -->
     <string name="google_maps_key" templateMergeStrategy="preserve" translatable="false">
-        YOUR_KEY_HERE
+        AIzaSyCbbFQd2dnMOT5SXuVEKhSwYVH-f0g-H5Y
```

Comment:
Ah, this is only for the release build (like to the Google Play store) - would remove and put back that placeholder